snapcraft/plugins/python2.py

```diff
@@ -168,6 +169,11 @@ def build(self):
              '--prefix={}/usr'.format(self.installdir),
              ], cwd=self.builddir)
 
+        # Fix all shebangs to use the in-snap python.
+        common.replace_in_file(self.installdir, re.compile(r''),
+                               re.compile(r'#!.*python'),
```

Comment:
Maybe we want to be clever here and not inadvertenly replace something we will regret?

A better match might be `r'#!{}.*python'.format(self.installdir)`, what do you think?